bitnami/rabbitmq/templates/svc.yaml

```
@@ -26,6 +26,9 @@ spec:
   {{- if .Values.service.loadBalancerSourceRanges }}
   loadBalancerSourceRanges: {{- toYaml .Values.service.loadBalancerSourceRanges | nindent 4 }}
   {{- end }}
+  {{- if .Values.service.allocateLoadBalancerNodePorts }}
+  allocateLoadBalancerNodePorts: {{ .Values.service.allocateLoadBalancerNodePorts }}
+  {{- end }}
```

This only happens when service is `LoadBalancer`, you will need to add a check here as done in L22.

---

Is this not nested within L22's if condition? Notice the {{- end }} at line 32

---

You are right! I missed the `{{-end} }` at line 32 :) Just a follow-up suggestion, I don't think we need the conditional here, .if `Values.service.allocateLoadBalancerNodePorts=false` then it's okay to have `allocateLoadBalancerNodePorts=false`  (once we have ensured the service type is LoadBalancer)

```suggestion
  allocateLoadBalancerNodePorts: {{ .Values.service.allocateLoadBalancerNodePorts }}
```